_dashboards/discover/dql.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: default
 title: Dashboards query language
-nav_order: 99
+parent: Discover
+nav_order: 1
```

Comment:
We'll need to add a redirect from the previous page, use this syntax:

```
redirect_from: 
  - /dashboards/dql/
```

Reply:
@Naarcha-AWS Done. Thanks for the review.